Gopkg.lock

```diff
@@ -68,7 +68,7 @@
 
 [[projects]]
   name = "github.com/go-kit/kit"
-  packages = ["endpoint","log","metrics","metrics/internal/lv","metrics/prometheus","transport/grpc","transport/http"]
+  packages = ["endpoint","log","log/level","metrics","metrics/internal/lv","metrics/prometheus","transport/grpc","transport/http"]
```

Comment:
I don't see this new package in the diff. Should Gopkg.lock changes be part of the commit?